vpr/src/route/connection_router.h

```diff
@@ -64,14 +69,18 @@ class ConnectionRouter {
         ::reset_path_costs(modified_rr_node_inf_);
     }
 
+    void empty_heap() {
+        heap_.empty_heap();
+    }
```

Comment:
Can you clarify why this needs to be exposed in the public interface?

Reply:
Done

Reply:
Done